Assets/MRTK/SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -68,8 +68,8 @@ public float ScaleLerpTime
         }
 
         [SerializeField]
-        [Tooltip("If true, the Solver will respect the object's original scale values")]
-        private bool maintainScale = true;
+        [Tooltip("If true, the Solver will respect the object's original scale values on initialization")]
```

Comment:
Should this have a `FormerlySerializedAs` tag?

Reply:
good catch, I'll make a followup PR to correct this